cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -4,15 +4,15 @@ NAME
 
 SYNOPSIS
        bal grpc [--input <proto-file-path>] [--output <path>]
```

Comment:
```suggestion
       bal grpc [--input] <proto-file-path> [--output] <path>
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/7748dedec94cfcd8b78279971effbc969478f5d5